ukrainian/old-photo/conspiracy/call/question/wtf.md

```diff
@@ -0,0 +1,9 @@
+Вам подобається музика Майкла Джексона?
+
+Людина по ту сторону слухавки відповідає, що не чула жодної композиції цього
+виконавця. Раптово голос говорить, що не може продовжити розмову та кладе слухавку. Через декілька хвилин задзвенів телефон, той же голос запитав чи подобається вам музика виконавця, якого ви зовсім не знаєте. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.